crates/detect-targets/Cargo.toml

```diff
@@ -10,7 +10,10 @@ edition = "2021"
 license = "Apache-2.0 OR MIT"
 
 [dependencies]
+futures-util = "0.3.28"
```

Comment:
I'd like to avoid this dep, tokio also supports tokio::join and tokio::try_join.